From f7bf06fae62bdf4383f7633208102b6576841a17 Mon Sep 17 00:00:00 2001
From: Hendrik Leppkes <h.leppkes@gmail.com>
Date: Tue, 3 Mar 2015 21:08:50 +0100
Subject: [PATCH 42/66] hevc: ignore overreads in the VPS, its not used in the
 decoding process

This allows some old clips to play with an incomplete VPS
---
 libavcodec/hevc_ps.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libavcodec/hevc_ps.c b/libavcodec/hevc_ps.c
index 217898c..f9d0e26 100644
--- a/libavcodec/hevc_ps.c
+++ b/libavcodec/hevc_ps.c
@@ -499,7 +499,6 @@ int ff_hevc_decode_nal_vps(HEVCContext *s)
     if (get_bits_left(gb) < 0) {
         av_log(s->avctx, AV_LOG_ERROR,
                "Overread VPS by %d bits\n", -get_bits_left(gb));
-        goto err;
     }
 
     if (s->vps_list[vps_id] &&
-- 
2.4.1

